add support for golang

Something like this would colorise the output of `grc go test -v ./..`:

```
regexp==== RUN .*
colour=blue
-
regexp=--- PASS: .*
colour=green
-
regexp=^PASS$
colour=green
-
regexp=^(ok|\?) .*
colour=magenta
-
regexp=--- FAIL: .*
colour=red
-
regexp=[^\s]+\.go(:\d+)?
colour=cyan
```

--- a/colourfiles/conf.go-test
+++ b/colourfiles/conf.go-test
@@ -1,0 +1,21 @@
+# go-test grc colorizer configuration
+regexp==== RUN .*
+colour=bright_blue
+-
+regexp=--- PASS: .* (\(\d+\.\d+s\))
+colour=green, yellow
+-
+regexp=^PASS$
+colour=bold white on_green
+-
+regexp=^(ok|FAIL)\s+.*
+colour=default, magenta
+-
+regexp=--- FAIL: .* (\(\d+\.\d+s\))
+colour=red, yellow
+-
+regexp=^FAIL$
+colour=bold white on_red
+-
+regexp=[^\s]+\.go(:\d+)?
+colour=cyan

--- a/grc.conf
+++ b/grc.conf
@@ -287,3 +287,7 @@ conf.lolcat
 (^|[/\w\.]+/)whois\s?
 conf.whois
 
+# go test
+(^|[/\w\.]+/)go test\s?
+conf.go-test
+

